setup.py

```diff
@@ -5,14 +5,14 @@
 from setuptools import find_packages, setup
 
 
-VERSION = "0.0.1"
+VERSION = "2.0.0"
```

Comment:
I think we should start with 0.1. Make it consistent with jaclang-jaseci. Since we haven't pushed to pypi yet, we can start low.

Reply:
I added a pyproject.toml file to use peotry for building this. So we should add the dependencies there and remove the setup.py